README.rst

```diff
@@ -0,0 +1,150 @@
+OpenMCyclus
+------------
+.. image:: https://github.com/arfc/openmcyclus/actions/workflows/test-openmcyclus.yml/badge.svg?branch=main
+
+Library of Cyclus archetypes to couple Cyclus with OpenMC.
+
+Installation 
+============
+
+Dependencies
+~~~~~~~~~~~~
+
+You will need to have `Cyclus <https://fuelcycle.org/>`_, `OpenMC <https://docs.openmc.org>`_, 
+and their required dependencies. Both are available via conda binaries or can 
+be installed from source via their github repositories. Please note 
+the versions listed below, as these are required versions, not minimum versions. 
+
++---------------+----------+
+| Dependency    | Version  |
++===============+==========+
+| Cyclus        | 1.6      |
++---------------+----------+
+| OpenMC        | 0.14     |
++---------------+----------+
+| SciPy         | 1.11     |
++---------------+----------+
+
+Directions to install Cyclus and OpenMC [1]_ 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is advised to create a new conda environment for installing all of the dependencies. 
+
+.. code-block:: bash
+  
+    ~ $ conda install -y gxx_linux-64 gcc_linux-64 cmake make git glib libxml2 libxmlpp-4.0 liblapack pkg-config coincbc boost-cpp hdf5 sqlite pcre setuptools pytest pytables pandas jinja2 cython websockets pprintpp pip mamba
+
+    ~ $ mamba install -y openmc=0.14.0 scipy=1.11
+
+    ~ $ git clone https://github.com/cyclus/cyclus.git
+
+    ~ $ cd cyclus
+
+    ~/cyclus $ python install.py
```

Comment:
What do you mean? These are the commands I put into my terminal. Is it because of the `~ $` formatting? I'm trying to show some of the directory structure for being inside the cyclus directory to install it. Do you have any suggestions for how it might be clearer?

Reply:
Yes, I am referring to the `~ $` formatting. If your intention is more to show how these commands would be in a terminal rather than have users copy and paste them, then this is fine.